ush/retrieve_data.py

```diff
@@ -48,12 +49,16 @@ def clean_up_output_dir(expected_subdir, local_archive, output_path, source_path
     output location."""
 
     unavailable = {}
+    expand_source_paths = []
+    for p in source_paths:
+        expand_source_paths.extend(glob.glob(p))
```

Comment:
Why was this modification made?

I'm finding that the fundamental `nco_grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_timeoffset_suite_GFS_v16` test is now failing in the make_ics/lbcs task because the necessary data isn't properly being pulled from HPSS.  This has been noted as failing on both Hera and Jet.  Removing lines 30, 52-55, and replacing `expand_source_paths` with the original `source_paths`, corrects the issue.